Sources/HAP/Endpoints/root().swift

```diff
@@ -26,7 +26,8 @@ func logger(_ application: @escaping Responder) -> Responder {
         let response = application(context, request)
         // swiftlint:disable:next line_length
         logger.info("\(context.channel.remoteAddress?.description ?? "N/A") \(request.method) \(request.urlString) \(response.status.code) \(response.body.count ?? 0)")
-        return response
+        logger.debug("- Response Messagea: \(String(data: response.body.data ?? "nil".data(using: .utf8)!, encoding: .utf8) ?? "-")\n")
```

Comment:
There's a typo and the `- ` prefix is IMO not needed. I think that this would generate non-printable output, especially when the initial pairing is going on. That might result in corrupting the tty session the server is running in, something that we should not do. We could show the http response headers and optionally, depending on the content type, the response's body.

Reply:
As I mentioned, this is putting back a logging line that has been present in the repository for a long time before the NIO integration, and preserves the original formating, including the spelling mistake from the old version ! I found it useful when searching logs, but we can indeed correct it. 
I would recommend keeping at least some kind of prefix, again for searching debug logs.

The `String(data:, encoding: .utf8)` initialiser returns nil if the unicode is not valid, so we get a `-` line in the log during pairing.

Reply:
> As I mentioned, this is putting back a logging line that has been present in the repository for a long time before the NIO integration, and preserves the original formating, including the spelling mistake from the old version !

I missed that, sorry!

> I found it useful when searching logs, but we can indeed correct it. I would recommend keeping at least some kind of prefix, again for searching debug logs.

Agree.

> The `String(data:, encoding: .utf8)` initialiser returns nil if the unicode is not valid, so we get a `-` line in the log during pairing.

Hmm ok. We could extend this to at least print the headers, but this is also fine I think.

Reply:
Fixed the Response log message prefix